examples/webvr_sandbox.html

```diff
@@ -163,6 +172,9 @@
 				mesh.rotation.x = time * 2;
 				mesh.rotation.y = time * 5;
 
+				box.position.setFromMatrixPosition(poseTarget.matrixWorld);
```

Comment:
What is this now? 😵

Reply:
Oh that's @brianpeiris remanent code from his PR. removing

Reply:
Oh, whoops. Thought I removed this myself. Thanks for cleaning up.